R/financial_functions.R

```diff
@@ -16,7 +16,37 @@
 #'   under baseline and late and sudden, market prices/costs, company net profit
 #'   margins, the proximity to target in the production forecast period and an
 #'   indication of the direction of the technology.
-calculate_net_profits <- function(data) {
+#' @param shock_year A numeric vector of length one that indicates in which year
+#'   the policy shock strikes in a given scenario.
+#' @param end_year Numeric, holding end year of analysis.
+#' @param carbon_data NGFS carbon prices.
+calculate_net_profits <- function(data,
+                                  shock_year,
+                                  end_year,
+                                  carbon_data) {
+
+  carbon_data <- carbon_data %>% dplyr::filter(.data$model == "MESSAGEix-GLOBIOM 1.0") %>% dplyr::select(-c(scenario_geography))
+  market_passthrough <- 0
+
+  data <-data %>%
```

Comment:
i would suggest to run the styler before merge the pr to get rid of these tiny formatting issues

Reply:
done